Content.Server/FloofStation/Traits/Components/ProductivePenisComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Shared.FixedPoint;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.FloofStation.Traits;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.List;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+using Robust.Shared.GameStates;
+
+namespace Content.Server.FloofStation.Traits;
+
+[RegisterComponent, Access(typeof(LewdTraitSystem))]
+public sealed partial class ProductivePenisComponent : Component
+{
+    [DataField("solutionname")]
+    public string SolutionName = "productivePenis";
+
+    [DataField]
+    public ProtoId<ReagentPrototype> ReagentId = "Cum";
+
+    [DataField]
+    public FixedPoint2 MaxVolume = FixedPoint2.New(50);
+
+    [DataField]
+    public Entity<SolutionComponent>? Solution = null;
+
+    [DataField]
+    public FixedPoint2 QuantityPerUpdate = 10;
+
+    [DataField]
+    public float HungerUsage = 10f;
+
+    [DataField]
+    public TimeSpan GrowthDelay = TimeSpan.FromSeconds(5);
+
+    [DataField(customTypeSerializer: typeof(TimeOffsetSerializer))]
+    public TimeSpan NextGrowth = TimeSpan.FromSeconds(0);
```

Comment:
NextGrowth doesn't seem to work properly and regenerate the solution, i don't know how Memeji got it to work for his

Reply:
You did not add the productiveCum to the update function in LewdTraitSystem.
Granted, this needs to be changed eventually as it is hardcoded, which is bad. For now though, add a query for the component in the update function.

Since it isn't in the update function, it doesn't know that it needs to regenerate the reagent.
Follow the queryCum while function that is in there.

Reply:
it works now! thankies!!

i dont really understand literally anything of what any of this means, i kind of just, change it around until it works

do you think the values as they are now are good? hopefully there's no "Cum Meta" in the future